plugins/discovery-gce/build.gradle

```diff
@@ -17,22 +17,23 @@ opensearchplugin {
   classname 'org.opensearch.plugin.discovery.gce.GceDiscoveryPlugin'
 }
 
-versions << [
-  'google': '1.23.0'
-]
-
 dependencies {
-  api "com.google.apis:google-api-services-compute:v1-rev160-${versions.google}"
-  api "com.google.api-client:google-api-client:${versions.google}"
+  api "com.google.apis:google-api-services-compute:v1-rev235-1.25.0"
+  api "com.google.api-client:google-api-client:1.35.2"
   api "com.google.oauth-client:google-oauth-client:1.35.0"
-  api "com.google.http-client:google-http-client:${versions.google}"
-  api "com.google.http-client:google-http-client-jackson2:${versions.google}"
+  api "com.google.http-client:google-http-client:${versions.google_http_client}"
+  api "com.google.http-client:google-http-client-gson:${versions.google_http_client}"
+  api "com.google.http-client:google-http-client-jackson2:${versions.google_http_client}"
   api 'com.google.code.findbugs:jsr305:3.0.2'
   api "org.apache.httpcomponents:httpclient:${versions.httpclient}"
   api "org.apache.httpcomponents:httpcore:${versions.httpcore}"
   api "commons-logging:commons-logging:${versions.commonslogging}"
   api "org.apache.logging.log4j:log4j-1.2-api:${versions.log4j}"
   api "commons-codec:commons-codec:${versions.commonscodec}"
+  api 'io.grpc:grpc-api:1.57.2'
+  api 'io.opencensus:opencensus-api:0.31.1'
```

Comment:
`io.opencensus` is suspicious, this library is abandoned (part of OTEL), why do we need it here?